MVC/models/api.model.js

```diff
@@ -0,0 +1,10 @@
+const db = require("../../db/connection.js");
+const apiEndpoints = require("../../endpoints.json");
+
+const fetchAPI = () => {
+  console.log(apiEndpoints);
+  console.log(typeof apiEndpoints);
+  return apiEndpoints;
+};
```

Comment:
model isn't really doing much. i'd just require in endpoints.json in the controller.